api/src/test/java/org/openmrs/module/ModuleUtilTest.java

```diff
@@ -730,4 +736,39 @@ protected File getEmptyJarDestinationFolder() throws IOException {
 		}
 		return destinationFolder;
 	}
+	
+	@Test
+	public void file2url_shouldReturnNullIfFileIsNull() throws MalformedURLException {
+
+		Assert.assertNull(ModuleUtil.file2url(null));
+	}
+	
+	@Test
+	public void getPackagesFromFile_shouldReturnEmptyStringSetIfNonJarFile() {
+		
+		File f = new File(this.getClass()
+			.getResource("/org/openmrs/module/include/test1-1.0-SNAPSHOT.omod").getFile());
+		Assert.assertTrue(ModuleUtil.getPackagesFromFile(f).isEmpty());
```

Comment:
you can use `assertThat(ModuleUtil.getPackagesFromFile(f), is(empty()))` using hamcrest matchers.

which will return the actual collection if it wasnt an empty string